src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -28,6 +28,7 @@
 	protected By patientHeaderId = By.cssSelector("div.identifiers span");
 	protected InitialSetupPage initialSetupPage;
 	protected PatientVisitsDashboardPage visitsDashboardPage;
+	protected HomePage homePage;
```

Comment:
Why including this page here

Reply:
Putting in mind that HomePage will be used by other workflows tests other than Location test, I think we can have it in the base class so that any test that will need it will access it through inheritance logic than declaring it in every test that will use it.

Reply:
Are you  calling it anywhere with in steps class?

Reply:
> Are you calling it anywhere with in steps class?

No

Reply:
You only need to include what is required i think

Reply:
@sherrif10 did you see the reason [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/131#discussion_r712758416) for declaring the page from the base class?

Reply:
its not worth maintaining it there, suppose its does not get used like that at all, Personally, i don't think we need to have that page initialised in steps class